samples/Calling/package.json

```diff
@@ -21,7 +21,6 @@
     "@azure/communication-identity": "1.0.0",
     "@azure/communication-calling": "1.1.0-beta.2",
     "@azure/communication-common": "1.0.0",
-    "@azure/communication-react": "1.0.0-beta.1",
```

Comment:
should this have been removed? (Ideally the sample _only_ references this package)

Reply:
It was a duplicate entry.